storage/plugins/archives/plugin.json

```
@@ -2,13 +2,14 @@
   "name": "archives",
   "display_name": "Archives (ISAD(G) / ISAAR(CPF))",
   "description": "Gestione di materiale archivistico e fotografico seguendo gli standard internazionali ISAD(G) (General International Standard Archival Description) e ISAAR(CPF) (authority record per persone, enti, famiglie). Modello gerarchico a 4 livelli — Fondo → Serie → Fascicolo → Unità. Ispirato al formato ABA (Arbejderbevægelsens Bibliotek og Arkiv, Copenhagen) e a ARKIS II degli Archivi Nazionali Svedesi.",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Biome (2.4.14)

[error] The key version was already declared.

(lint/suspicious/noDuplicateObjectKeys)